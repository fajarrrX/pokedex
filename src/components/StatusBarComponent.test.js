import React from "react";
import renderer from "react-test-renderer";
import StatusBarComponent from "./StatusBarComponent";

it("StatusBarComponent screen renders correctly", () => {
  renderer.create(<StatusBarComponent />);
});

test("StatusBarComponent snapShot", () => {
  const snap = renderer.create(<StatusBarComponent />).toJSON();
  expect(snap).toMatchSnapshot();
});
