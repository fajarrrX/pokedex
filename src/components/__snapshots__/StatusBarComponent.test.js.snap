// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBarComponent snapShot 1`] = `
<View
  style={
    Object {
      "backgroundColor": undefined,
      "height": 30,
    }
  }
/>
`;
